### ISSUE TITLE
Add XML output option

### ISSUE THREAD

**Issue by ajrosen** [ajrosen/icalPal#11]
https://github.com/ajrosen/icalPal/issues/10#issuecomment-2038301957

`
I wonder how tricky it would be to add an xml output option? I'm trying to streamline a workflow using a particular WP import plugin that only accepts csv and xml files.
`

**Comment by ajrosen** [comment]
@selasphorus

For your use case, how "official" does the XML need to be?  Is a simple series of `<field>value</field>` tokens enough?

```
<events>
  <event>
    <account>iCloud</account>
    <type>Subscribed</type>
    <color>#FCB80EFF</color>
    <calendar>US Holidays</calendar>
    <subcal_url>https://calendars.icloud.com/holidays/us_en-us.ics/</subcal_url>
    <symbolic_color_name>custom</symbolic_color_name>
    <end_date>694223999</end_date>
    <orig_date/>
    <start_date>694137600</start_date>
    <duration>86399</duration>
    <all_day>1</all_day>
    <availability>1</availability>
    <conference_url_detected/>
    <notes/>
    <has_recurrences>1</has_recurrences>
    <invitation_status>0</invitation_status>
    <orig_item_id>0</orig_item_id>
    <ROWID>16680</ROWID>
    <start_tz>_float</start_tz>
    <status>0</status>
    <title>New Year’s Eve</title>
    <unique_identifier>87467be6-af75-3768-a987-0a11536ae0b4</unique_identifier>
    <url/>
    <UUID>AD35DE1A-00E5-4321-99CD-EC4DECC52546</UUID>
    <xdate/>
    <attendees/>
    <address/>
    <location/>
    <count>6</count>
    <rend_date/>
    <frequency>4</frequency>
    <interval>1</interval>
    <specifier/>
    <trigger_interval/>
    <edate>Dec 31, 2023</edate>
    <sdate>Dec 30, 2023</sdate>
    <stime>1703991600</stime>
    <etime>1704077999</etime>
    <sday>Dec 30, 2023</sday>
  </event>
  <event>
    <account>iCloud</account>
    <type>Subscribed</type>
    <color>#FCB80EFF</color>
    <calendar>US Holidays</calendar>
    <subcal_url>https://calendars.icloud.com/holidays/us_en-us.ics/</subcal_url>
    <symbolic_color_name>custom</symbolic_color_name>
    <end_date>662774399</end_date>
    <orig_date/>
    <start_date>662688000</start_date>
    <duration>86399</duration>
    <all_day>1</all_day>
    <availability>1</availability>
    <conference_url_detected/>
    <notes/>
    <has_recurrences>1</has_recurrences>
    <invitation_status>0</invitation_status>
    <orig_item_id>0</orig_item_id>
    <ROWID>16684</ROWID>
    <start_tz>_float</start_tz>
    <status>0</status>
    <title>New Year’s Day</title>
    <unique_identifier>00069f75-a269-3014-8ecb-7b7c2e469e5b</unique_identifier>
    <url/>
    <UUID>89037754-8E17-4497-BE45-9EF05C1F8573</UUID>
    <xdate/>
    <attendees/>
    <address/>
    <location/>
    <count>6</count>
    <rend_date/>
    <frequency>4</frequency>
    <interval>1</interval>
    <specifier/>
    <trigger_interval/>
    <edate>Jan 1, 2024</edate>
    <sdate>Dec 31, 2023</sdate>
    <stime>1704078000</stime>
    <etime>1704164399</etime>
    <sday>Dec 31, 2023</sday>
  </event>
</events>
```

**Comment by ajrosen** [comment]
@selasphorus

Or if anyone would like to test it, I've published a beta release.

https://github.com/ajrosen/icalPal/releases/download/1.3.0-beta1/icalPal-1.3.0.gem

**Comment by ajrosen** [comment]
Included in version 2.0.0

**Comment by selasphorus** [comment]
Thank you! It's working pretty well. I'm going to open a fresh issue re the notes field (and potentially other fields which might contain line breaks).
